382 remove redundant isPartOf statements

ChristophEwertowski opened the pull request:
Resolves #382

dr0i commented on src/main/resources/morph-hbz01-to-lobid.xml:
```diff
@@ -1871,7 +1867,8 @@
 					<none>
-						<data source="@isPartOfHbzId"/>
+						<data source="4[56789]3[-abr][-12].a"/>
+						<!-- <data source="@isPartOfHbzId"/> -->
```
Also, better remove unused code.

dr0i commented on src/main/resources/morph-hbz01-to-lobid.xml:
```diff
@@ -224,12 +224,11 @@
 			<regexp match="(.*)" format="$[ns-lobid-resource]${1}#!"/>
 			<sanitizeUrl></sanitizeUrl>
-		</data>
+		</data> -->
```
Better remove unused rules.

dr0i commented on src/test/resources/jsonld/HT015891797:
```diff
@@ -74,13 +74,6 @@
-  }, {
-    "id" : "http://lobid.org/resources/HT002786685#!",
-    "label" : "lobid Ressource"
-  } ],
```
Is it correct to loose the link to HT002786685 ? Can't find it redundantly residing in an other field.

dr0i commented on src/test/resources/jsonld/HT016132962:
```diff
@@ -0,0 +1,67 @@
+  } ],
+  "title" : "Konferenz \"Supply Chain Management im Mittelstand\"",
+  "type" : [ "BibliographicResource", "Book" ]
+}
```
Why adding this resource to the test set? Does it test something that is not already tested by an existing test document?

dr0i commented on src/test/resources/jsonld/HT019268984:
```diff
@@ -0,0 +1,114 @@
+    "numbering" : "1",
+    "type" : [ "MultiVolumeWorkRelation" ]
+  } ]
+}
```
Why adding this resource to the test set? Does it really test something that is not already tested by an existing test document?

dr0i commented on src/test/resources/output/json/00196/TT001966831.json:
```diff
@@ -37,13 +37,6 @@
-  }, {
-    "id" : "http://ld.zdb-services.de/resource/626580-7",
-    "label" : "http://ld.zdb-services.de/resource"
-  } ],
```
Here we loose the connection to ld.zdb-services.de/resource/626580-7 whih is not redundantly covered by another field. Is this intended?

dr0i commented on src/test/resources/output/json/00216/TT002165019.json:
```diff
@@ -0,0 +1,57 @@
+  } ],
+  "title" : "Bauen wir doch aufs neue das alte Haus",
+  "type" : [ "BibliographicResource", "Book" ]
+}
```
Is this accidentally added ? This resource has item informations that are not yet part of it's the parent resource.

dr0i replied:
Ah -forget my comment about the item infos. They reside here as for all the other resources because this JSON-LD file does not reflect the ES index.

dr0i commented on src/test/resources/output/json/00594/HT005944358.json:
```diff
@@ -0,0 +1,105 @@
+    "numbering" : "95",
+    "type" : [ "MultiVolumeWorkRelation" ]
+  } ]
+}
```
Is this accidentally added ? ~~This resource has item informations that are not yet part of it's the parent resource.~~

dr0i commented on src/test/resources/output/json/01299/HT012996101.json:
```diff
@@ -49,41 +49,26 @@
   } ],
   "subject" : [ {
-    "componentList" : [ {
-      "id" : "http://d-nb.info/gnd/4061778-6",
```
Why is the componentList removed?

ChristophEwertowski commented:
* Deleted HT016132962 and HT019268984 because the test cases (redundant isPartOf from inSeries/inVolume) are also covered by, for example, CT003012479/BT000003404.
* Removed commented code.
* It is correct to lose the link in HT002786685 and TT001966831 because the information doesn't come from observation (```ind2="2"```)
* To do: Look at the removed component list, resolve conflicts, look if ZDB-resources are covered.

dr0i commented:
Deployed to production, see e.g. http://lobid.org/resources/HT015891797?format=json. Closed.
